examples/nlp/gpt/train_gpt_reinforce_actor.py

```diff
@@ -0,0 +1,198 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Do we want to update the copyright date?

Reply:
@ko3n1g can this be automated? at least the correctness && presence of this header?

Reply:
I'll resolve this since @ko3n1g is aware and working on it in parallel